### PR TITLE
Added version validation to added files

### PR DESCRIPTION
**Pull request by yardensade**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15253

### CODE REVIEW

**Comment by ronykoz** [comment]
Please add validation for json files as well

**Comment by yardensade** [comment]
Done, but seems like not every json file has version, so I only test it's file version if it has version key.

File: Tests/scripts/validate_files_structure.py [hunk 1]
```diff
@@ -382,14 +382,21 @@ def changed_docker_image(file_path):
...
-    if is_incorrect_version or is_incorrect_version_secondary:
+    file_extension = os.path.splitext(file_path)[1]
+    version_number = -1
+    if file_extension == '.yml':
+        yaml_dict = get_json(file_path)
+        version_number = yaml_dict.get('commonfields', {}).get('version')
+    elif file_extension == '.json':
+        with open("./" + file_path) as json_file:
+            json_dict = json.load(json_file)
+            if isinstance(json_dict, 'dict'):
```
Comment (Itay4) [inline Tests/scripts/validate_files_structure.py]: what if it isn't `dict` ?

File: Tests/scripts/validate_files_structure.py [hunk 2]
```diff
@@ -382,14 +382,21 @@ def changed_docker_image(file_path):
...
+                if 'version' in json_dict:
```
Comment (Itay4) [inline Tests/scripts/validate_files_structure.py]: how come `version` will not exist ?

File: Tests/scripts/validate_files_structure.py [hunk 3]
```diff
@@ -514,7 +521,7 @@ def validate_added_files(added_files, integration_set, playbook_set, script_set,
     has_schema_problem = False
     for file_path in added_files:
         print "Validating {}".format(file_path)
-        if not validate_schema(file_path):
+        if not validate_schema(file_path) or validate_version(file_path):
```
Comment (Itay4) [inline Tests/scripts/validate_files_structure.py]: shouldn't it be `not validate_version(file_path)` ?

Comment (yardensade) [reply Tests/scripts/validate_files_structure.py]: nope it returns true if it has validation issue

File: Tests/scripts/validate_files_structure.py (hunk 2 above)
Comment (yardensade) [reply Tests/scripts/validate_files_structure.py]: Not every json file has version from what i've been looking at the repo

File: Tests/scripts/validate_files_structure.py (hunk 1 above)
Comment (yardensade) [reply Tests/scripts/validate_files_structure.py]: If it isn't a dict it probably does not have versioning that needs checkup, from what I saw in the repo.

**Pull request merged**